src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -472,6 +472,7 @@
 
         :else
         {:status 200
+         :headers {"Content-Type" "application/octet-stream"}
```

Comment:
This solves the immediate problem of setting the content type for a successful request. However, there are a lot of other cases where we don't return a content type - such as the error cases prior to this line. Should we be setting a default content-type of `text/plain` if no content-type was given?